gapic/api/artman_language_v1.yaml

```diff
@@ -6,46 +6,65 @@ common:
   proto_deps:
     - google-common-protos
   import_proto_path:
-    - ${REPOROOT}/googleapis
+    - ${GOOGLEAPIS}
   src_proto_path:
-    - ${REPOROOT}/googleapis/google/cloud/language/v1
+    - ${GOOGLEAPIS}/google/cloud/language/v1
   service_yaml:
-    - ${REPOROOT}/googleapis/google/cloud/language/language_v1.yaml
+    - ${GOOGLEAPIS}/google/cloud/language/language_v1.yaml
   output_dir: ${REPOROOT}/artman/output
+  git_repos:
+    staging:
+      location: git@github.com:googleapis/api-client-staging.git
   gapic_api_yaml:
-    - ${REPOROOT}/googleapis/google/cloud/language/v1/language_gapic.yaml
+    - ${GOOGLEAPIS}/google/cloud/language/v1/language_gapic.yaml
 csharp:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-cloud-language
 go:
   gapic_code_dir: ${REPOROOT}/gapi-cloud-language-go
-  git_repo:
-    location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
-    gapic_subpath: language/apiv1
+  git_repos:
+    go:
+      location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
+      gapic_subpath: language/apiv1
+    staging:
+      gapic_subpath: google-cloud-language-v1/cloud.google.com/go/language/apiv1
 java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-language
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-language-v1
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-language
+  git_repos:
+    java:
+      gapic_component: src/main/java/com/google/cloud/language/spi/v1
+      gapic_subpath: google-cloud-pubsub/src/main/java/com/google/cloud/language/spi/v1
```

Comment:
This should be `google-cloud-language` (it looks like you have `google-cloud-pubsub` in a number of places)